Convert remaining SupportPower uses of seconds to ticks

reaperrr opened the pull request:
Logn overdue, in my opinion.

reaperrr committed:
Refactor support powers to use ticks instead of seconds for all delays - code changes

Also renamed ChargeTime to ChargeInterval.

reaperrr committed:
Upgrade RA rules to SupportPower seconds -> ticks

reaperrr committed:
Upgrade TD rules to SupportPower seconds -> ticks

reaperrr committed:
Upgrade D2k rules to SupportPower seconds -> ticks

reaperrr committed:
Upgrade TS rules to SupportPower seconds -> ticks

pchote commented on OpenRA.Mods.Cnc/Traits/SupportPowers/GpsPower.cs:
```diff
@@ -22,6 +22,7 @@ namespace OpenRA.Mods.Cnc.Traits
 	[Desc("Requires `GpsWatcher` on the player actor.")]
 	class GpsPowerInfo : SupportPowerInfo
 	{
+		[Desc("Delay of map reveal in ticks.")]
```
"Delay in ticks between launching and revealing the map"?

pchote commented on OpenRA.Mods.Cnc/Traits/SupportPowers/IonCannonPower.cs:
```diff
@@ -24,7 +24,7 @@ class IonCannonPowerInfo : SupportPowerInfo, IRulesetLoaded
 		public readonly string CameraActor = null;
 
-		[Desc("Amount of time to keep the camera alive")]
+		[Desc("Amount of ticks to keep the camera alive")]
```
"Number of ticks to keep the camera alive"

pchote commented on OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs:
```diff
@@ -15,8 +15,8 @@ namespace OpenRA.Mods.Common.Traits
-		[Desc("Measured in seconds.")]
-		public readonly int ChargeTime = 0;
+		[Desc("Measured in ticks.")]
+		public readonly int ChargeDelay = 0;
```
Can we please call this `Interval` instead? Our naming convention is to use *Delay for one-off events, and Interval for things that repeat.

reaperrr commented:
Updated.

MustaphaTR commented:
Needs rebase.

reaperrr commented:
Rebased.

reaperrr committed:
Remove redundant custom Duration from Bomber John Chronoshift power

The MNLYR has ReturnToOrigin: false anyway, so this was redundant and made no difference whatsoever.

reaperrr commented:
Minor update (see last commit) and rebased.

GraionDilach reviewed (approved):
:+1:

pchote commented:
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/44ca09a04849408435d73a3838f502495c7fab6e
